ccx_messaging/downloaders/idp_downloader.py

```diff
@@ -0,0 +1,47 @@
+# Copyright 2024 Red Hat Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""S3 Downloader for internal data pipeline."""
+
+from insights_messaging.downloaders.s3 import S3Downloader as S3Downloader
+class IDPDownloader(S3Downloader):
```

Comment:
I would name it with a more generic name, like `S3Downloader`, so what I would do is to import `S3Downloader` as `ICMS3Downloader` and then name this one as the later.